java/mykola-melnyk/src/com/zoolatech/lecture4/tasks/_2/BinarySearchTree.java

```diff
@@ -0,0 +1,44 @@
+package com.zoolatech.lecture4.tasks._2;
+
+public class BinarySearchTree {
+    private Node rootNode;
+    public BinarySearchTree () {
+        rootNode = null;
```

Comment:
the default value of rootNode is null, you can remove both the constructor and initialization statement

Reply:
Done